nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md

```diff
@@ -133,6 +133,8 @@ Again, try swapping the cluster back and forth, to ensure it can move freely bet
 
 ## Setting up the workers
 
+NOTE: As of Version 6 NServiceBus no longer supports running an embedded distributor. Because of that configuration sections like `MasteNodeConfig` are obsoleted and `endpointConfiguraiton.EnlistWithLegacyMSMQDistributor` should be used instead. Read more about [Upgrading a Distributor-based scaled out endpoint to V6](/samples/scaleout/distributor-upgrade/)
```

Comment:
@WojcikMike - I think this should also be in the upgrade guide. Also is the distributor not relevant for v6 anymore @SzymonPobiega? If so, I think we should just add a general WARNING note at the top of this article.  Thoughts?

Also when using versions, use Versions 6 and above or Versions 5 and below. See https://github.com/particular/docs.particular.net#version-language